pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -984,7 +987,7 @@ func (m *ManagerImpl) callPreStartContainerIfNeeded(podUID, contName, resource s
 
 	if eI.opts == nil || !eI.opts.PreStartRequired {
 		m.mutex.Unlock()
-		klog.V(4).InfoS("Plugin options indicate to skip PreStartContainer for resource", "resourceName", resource)
+		klog.V(5).InfoS("Plugin options indicate to skip PreStartContainer for resource", "podUID", podUID, "resourceName", resource, "containerName", contName)
```

Comment:
the same here. Are there any guidelines regarding this?

Reply:
For the same reasons as mentioned earlier: minimizing noise and maintaining consistency across resource managers. Please refer to the previous CPU Manager logging improvement for context: https://github.com/kubernetes/kubernetes/pull/120569/commits/2dc5ddd08a41a171134fd0922825ed03faa379ca.

Unfortunately, I haven't found guidelines/recommendations with this level of details in Kubernetes documentation. I know @pohly is quiet active in this area and might know?

I have been drafting some guidelines as I am working through the resource managers and am happy to contribute this for future reference but I am not sure where? :)

Reply:
I've seen some guidelines made by @pohly in the DRA code, but I don't remember where exactly. @pohly can you help?

Reply:
Guidelines for logging levels? https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#what-method-to-use might be what you look for.

It's up to the code owners to decide which things are useful enough to be logged more often.

> I've seen some guidelines made by @pohly in the DRA code, but I don't remember where exactly. @pohly can you help?

Perhaps https://github.com/kubernetes/kubernetes/blob/814e2c89a80958bb0ba0ff186a41cda32cfe06da/staging/src/k8s.io/dynamic-resource-allocation/structured/allocator.go#L102-L106?

Reply:
Perfect, Thank you!